lightning/src/ln/channelmanager.rs

```diff
@@ -118,9 +118,16 @@ pub(super) enum PendingHTLCStatus {
 
 pub(super) enum HTLCForwardInfo {
 	AddHTLC {
+		forward_info: PendingHTLCInfo,
+
+		// These fields are produced in `forward_htlcs()` and consumed in
+		// `process_pending_htlc_forwards()` for constructing the
+		// `HTLCSource::PreviousHopData` for failed and forwarded
+		// HTLCs.
 		prev_short_channel_id: u64,
 		prev_htlc_id: u64,
-		forward_info: PendingHTLCInfo,
+		prev_channel_outpoint: OutPoint,
```

Comment:
Can we refer to these as the `funding_outpoint` - `outpoint` by itself is a bit confusing.